assets/docs/configuration/targets/http-full-example.hcl

```
@@ -71,9 +71,12 @@ target {
     # Optional path to the file containing template which is used to build HTTP request based on a batch of input data
     template_file              = "myTemplate.file"
 
-    # Optional. When enabled, configured client timeout and timestamp of request are attached as HTTP headers.
+    # Optional. When enabled, precalculated rejection timestamp is attached as HTTP header. This can be used as a hint for target server when to reject incoming HTTP request. 
```

```suggestion
    # Optional. When enabled, precalculated rejection timestamp is attached as HTTP header. This option is part of a feature of the Snowplow Event Forwarding product.
```

^^ I don't expect anyone to use this feature if it's not part of our product, so just a slight rephrase here.